### PR TITLE
changed layout + img ready design

### PR DESCRIPTION
**Pull request by Quantumf0x**

### CODE REVIEW

**Comment by flkremer** [comment]
...

File: MemoryGame-1B/Views/NewGame.xaml [hunk 1]
```diff
@@ -8,55 +8,98 @@
       d:DesignHeight="450" d:DesignWidth="800"
       Title="NewGame">
     <Grid>
+        <Grid.Background>
+            <ImageBrush ImageSource="background.jpg"/>
```
Comment (JipSterk) [inline MemoryGame-1B/Views/NewGame.xaml]: Name is inconsistent please make it with a capital b also where is background.jpg located?

File: MemoryGame-1B/Views/NewGame.xaml [hunk 2]
```diff
@@ -8,55 +8,98 @@
...
-                        <RowDefinition Height="60*"></RowDefinition>
-                        <RowDefinition Height="60*"></RowDefinition>
+                        <RowDefinition Height="20*"></RowDefinition>
                     </Grid.RowDefinitions>
+
                     <Grid Column="1"  Row="1">
                         <Grid.RowDefinitions>
-                            <RowDefinition Height="30*"></RowDefinition>
-                            <RowDefinition Height="30*"></RowDefinition>
+                            <RowDefinition  Height="40*"></RowDefinition>
```
Comment (JipSterk) [inline MemoryGame-1B/Views/NewGame.xaml]: unnecessary white space

File: MemoryGame-1B/Views/NewGame.xaml [hunk 3]
```diff
@@ -8,55 +8,98 @@
...
+                            <RowDefinition Height="40*"></RowDefinition>
                         </Grid.RowDefinitions>
-                        <TextBlock Grid.Row="0" Margin="30 0 0 0">Playername1</TextBlock>
-                        <TextBlock Grid.Row="1" Margin="30 0 0 0">score 0</TextBlock>
+                        <Canvas Width="{Binding ElementName=gd, Path=ActualWidth}"
```
Comment (JipSterk) [inline MemoryGame-1B/Views/NewGame.xaml]: what is gd here please use a more descriptive name like ElementName

File: MemoryGame-1B/Views/NewGame.xaml [hunk 4]
```diff
@@ -8,55 +8,98 @@
...
+                            </Viewbox>
+                            <Viewbox HorizontalAlignment="Left" Margin="30,0,0,0" Width="130" Canvas.Top="70" Canvas.Left="30">
+                                <TextBlock  Grid.Row="0" MinWidth="130">Playername2</TextBlock>
+                            </Viewbox>
+                            <Viewbox Grid.Row="1" HorizontalAlignment="Left" Margin="30 0 0 0" Canvas.Top="90" Canvas.Left="30">
+                                <TextBlock Grid.Row="1" MinWidth="130">score 0</TextBlock>
+                            </Viewbox>
+                            <!-- <TextBlock Text="My text" Canvas.Top="30" Canvas.Left="30" ></TextBlock> -->
+                        </Canvas>
+                        <!-- <Viewbox HorizontalAlignment="Left" Margin="30 0 0 0" Width="130"> -->
```
Comment (JipSterk) [inline MemoryGame-1B/Views/NewGame.xaml]: Remove commented code

File: MemoryGame-1B/Views/NewGame.xaml [hunk 5]
```diff
@@ -8,55 +8,98 @@
...
-                            <RowDefinition Height="30*"></RowDefinition>
-                            <RowDefinition Height="30*"></RowDefinition>
+                            <RowDefinition Height="40*"></RowDefinition>
+                            <RowDefinition Height="40*"></RowDefinition>
                         </Grid.RowDefinitions>
-                        <TextBlock Grid.Row="0" Margin="30 0 0 0">Playername2</TextBlock>
-                        <TextBlock Grid.Row="1" Margin="30 0 0 0">score 0</TextBlock>
+
+
+                        <!-- <Viewbox HorizontalAlignment="Left" Margin="30,0,0,0" Width="130"  > -->
```
Comment (JipSterk) [inline MemoryGame-1B/Views/NewGame.xaml]: remove commented code

File: MemoryGame-1B/Views/NewGame.xaml [hunk 6]
```diff
@@ -8,55 +8,98 @@
...
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="40*"/>
+                        <ColumnDefinition Width="40*"/>
+                        <ColumnDefinition Width="40*"/>
+                    </Grid.ColumnDefinitions>
                     <Grid.RowDefinitions>
                         <RowDefinition Height="60*"></RowDefinition>
-                        <RowDefinition Height="60*"></RowDefinition>
-                        <RowDefinition Height="60*"></RowDefinition>
+                        <!-- <RowDefinition Height="60*"></RowDefinition> -->
```
Comment (JipSterk) [inline MemoryGame-1B/Views/NewGame.xaml]: remove commented code

File: MemoryGame-1B/Views/NewGame.xaml [hunk 7]
```diff
@@ -8,55 +8,98 @@
...
+                        <!-- <RowDefinition Height="60*"></RowDefinition> -->
                     </Grid.RowDefinitions>
-                    <Button Grid.Row="0" Margin="30, 10" Click="Save">Save</Button>
-                    <Button Grid.Row="1" Margin="30, 10" Click="ShowMenu">Menu</Button>
-                    <Button Grid.Row="2" Margin="30, 10" Click="RestartGame">Restart game</Button>
+                    <Button Grid.Column="0"  Margin="10" Click="Save">Save</Button>
```
Comment (JipSterk) [inline MemoryGame-1B/Views/NewGame.xaml]: remove whitespace

File: MemoryGame-1B/Views/NewGame.xaml (hunk 3 above)
Comment (Quantumf0x) [reply MemoryGame-1B/Views/NewGame.xaml]: Removed doesn't work

**Pull request closed without merging**